docs/src/pages/style/typography/Types.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles({
```

Comment:
I've been using createStyles only when the compiler is having issues with it and in most cases I haven't needed to use it 😄 For consistency it might be good to use it for this documentation like you suggest.

Reply:
Updated to include `createStyles()`